src/IdBrokerClient.php

```diff
@@ -434,6 +434,41 @@ public function mfaVerify(string $id, string $employeeId, $value, string $rpOrig
         $this->reportUnexpectedResponse($result, 1506710704);
     }
 
+    /**
+     * Verify an MFA registration (only for webauthn)
+     * @param string $id The MFA ID.
+     * @param string $employeeId The Employee ID of the user with that MFA.
+     * @param string|array $value The MFA value being verified.
+     * @param string $rpOrigin The Relying Party Origin, for WebAuthn MFA options.
+     * @return bool|array
+     * @throws MfaRateLimitException
+     * @throws ServiceException
+     */
+    public function mfaVerifyRegistration(string $id, string $employeeId, $value, string $rpOrigin = '')
+    {
+        $result = $this->mfaVerifyInternal([
+            'id' => $id,
+            'type' => 'registration',
```

Comment:
Could you have used the existing `mfaVerify` function with an added `type` argument?

Reply:
@forevermatt How do you prefer doing this in Yii. I read something that said that Yii doesn't allow optional url path params in the frontend config.